site/src/examples/overlay/WithActions.tsx

```diff
@@ -24,13 +24,10 @@ const checkboxesData = [
   },
 ];
 
-const WithActionsContent = ({
-  onClose,
-  id,
-}: {
-  onClose: () => void;
-  id: string;
-}) => {
+const WithActionsContent = (
```

Comment:
Something is wrong in this example: 
`<h3 id="[object Object]"`

Reply:
fixed